### PR TITLE
fix(ipAutozoomControl): should not need to check az status when using…

### PR DESCRIPTION
**Pull request by pkhansen**
… enable/disable

### CODE REVIEW

File: src/components/ipAutozoomControl.ts [hunk 1]
```diff
@@ -95,7 +89,7 @@ export default class IpAutozoomControl implements IAutozoomControl {
     const cnnFeature = new huddly.CnnFeature();
     cnnFeature.setFeature(feature);
     const azStatus = await this._deviceManager.getCnnFeatureStatus(cnnFeature);
-    return azStatus.getEnabled();
+    return azStatus.getAzStatus().getAzEnabled();
```
Comment (eskilgh) [inline src/components/ipAutozoomControl.ts:92]: Is this the weird case when you have a AzStatus inside CnnStatus or something?

**Review by eskilgh — APPROVED** [review]
Always great to remove code 👏  Any idea why this was there in the first place?

File: src/components/ipAutozoomControl.ts (hunk 1 above)
Comment (pkhansen) [reply src/components/ipAutozoomControl.ts:92]: Yes, the only way it was done before. At some point I thought it was backwards compatible ☹️

**Comment by pkhansen** [comment]
> Always great to remove code 👏 Any idea why this was there in the first place?

Likely to match the usb implementation way back

**Pull request merged**